lib/matplotlib/lines.py

```diff
@@ -1250,9 +1244,6 @@ def set_dashes(self, seq):
         else:
             self.set_linestyle((0, seq))
 
-    def _draw_lines(self, renderer, gc, path, trans):
```

Comment:
The `_drawStyles_l` dictionary still references `_draw_lines`, and so I would imagine that by removing this method would cause problems (but I am not sure I am clear how it would have worked properly in the first place?).